doc/man3/SSL_CTX_set_options.pod

```diff
@@ -217,6 +217,18 @@ not propose, and servers will not accept the extension.
 Disable all renegotiation in TLSv1.2 and earlier. Do not send HelloRequest
 messages, and ignore renegotiation requests via ClientHello.
 
+=item SSL_OP_IGNORE_UNEXPECTED_EOF
+
+Some TLS implementations do not send a close_notify alert on shutdown. Before
+OpenSSL 3.0, B<SSL_ERROR_SYSCALL> with errno=0 was reported in such cases. This
+behaviour contradicts the B<SSL_ERROR_SYSCALL> documentation. Since OpenSSL 3.0
+the returned error is B<SSL_ERROR_SSL> with a meaningful error on the error
+stack. When this option is enabled an B<SSL_ERROR_ZERO_RETURN> error is
+returned in case an EOF is detected and the close_notify alert was not
+received. You should only enable this option if the protocol running over TLS
+can detect a truncation attack itself, and that the application is checking for
+that truncation attack.
```

Comment:
This is missing the changes you did in ignore_unexpected_eof above

Reply:
Sorry, could you please specify the exact place to inherit the changes from?

Reply:
The one in SSL_CONF_cmd.pod

Reply:
Done. Thanks!